plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -53,6 +53,17 @@ def extract_rpm_database
         files[pair.first] = pair[1]
       end
     end
+    # make sure that all parent directories of managed rpm directories are considered
+    # managed
+    dirh.dup.each do |d, _e|
+      dir, _sep, _file = d.rpartition("/")
```

Comment:
I would do `dir = d.rpartition("/").first`, then you don't need unused variables.